test/e2e-v2/cases/apisix/otel-collector/docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
     networks:
       e2e:
   otel-collector:
-    image: otel/opentelemetry-collector:0.50.0
+    image: otel/opentelemetry-collector:0.92.0
```

Comment:
Let's make this version as an env variable.  Can refer to：https://github.com/apache/skywalking/blob/a9128e0cefbcba08c085b70ed8795d7fb416459c/test/e2e-v2/cases/storage/es/docker-compose.yml#L20

Reply:
Done

Reply:
@peachisai I think you could set a global env variable after here for the whole  e2e-test job https://github.com/apache/skywalking/blob/a9128e0cefbcba08c085b70ed8795d7fb416459c/.github/workflows/skywalking.yaml#L332

``` yaml
    env:
      OTEL_COLLECTOR_VERSION: 0.92.0
```

Reply:
> @peachisai I think you could set a global env variable after here for the whole e2e-test job
> 
> https://github.com/apache/skywalking/blob/a9128e0cefbcba08c085b70ed8795d7fb416459c/.github/workflows/skywalking.yaml#L332
> 
> ```yaml
>     env:
>       OTEL_COLLECTOR_VERSION: 0.92.0
> ```

Okay,  I originally had in mind  we can assign values individually for each test.